src/esp/assets/ResourceManager.h

```diff
@@ -337,13 +436,63 @@ class ResourceManager {
   int getNumLibraryObjects() { return physicsObjTemplateLibrary_.size(); };
 
   /**
-   * @brief Retrieve the composition of all transforms applied to a mesh since
-   * it was loaded.
+   * @brief Gets the number of loaded object templates stored in the @ref
+   * physicsObjTemplateLibrary_.
+   *
+   * @return The number of entries in @ref physicsObjTemplateLibrary_ that are
+   * loaded from files.
+   */
+  int getNumFileTemplateObjects() { return physicsObjTmpltLibByID_.size(); };
+  /**
+   * @brief Get a random loaded attribute handle for the loaded file-based
+   * object templates
+   *
+   * @return a randomly selected handle corresponding to the a file-based object
+   * attributes template, or empty string if none loaded
+   */
+  std::string getRandomFileTemplateHandle() {
```

Comment:
This distinction seems unnecessary after re-design. I understand the separate storage/query of a "primitive template" and "physics object template", but a file origin should not matter anymore.

Reply:
I felt we still wished to distinguish between objects that were synthesized primitives and objects that were file-based, at least until the primitive object instancing has been completed (i.e. for a debugging tool).  It's only a shallow distinction - all the underlying code treats the object templates the same.